### ISSUE TITLE
Request: Sublime Merge theme

### ISSUE THREAD

**Issue by hsiktas** [Monokai/monokai-pro-sublime-text#98]
[Sublime Merge](https://www.sublimemerge.com/) has been out for a while now and I would guess that the majority of its users are also Sublime Text users. Would it be possible to add a variant of this theme also for Sublime Merge so that we can have matching themes between ST and SM?
Themes support seems to be similar to Sublime Text: https://www.sublimemerge.com/docs/themes

Imho, it would be ok to sell another license for a SM variant. 😉

**Comment by DanielPower** [comment]
I'd also be willing to buy a separate license for a Sublime Merge theme.

**Comment by rmmullen** [comment]
Same^

**Comment by ivansvlv** [comment]
Agree, I have both VSCode and Sublime licenses, wouldn't hesitate for Sublime Merge as well.

**Comment by lzaaldian** [comment]
Same here. I have both VS Code and Sublime licenses and certainly want the Sublime Merge version 🙌

**Comment by lzaaldian** [comment]
> Here is [something](https://github.com/Resike/Merge-Monokai-Theme) for you guys.

Awesome! One step closer to make this happens. Hope the author will release the official theme.

**Comment by bitsper2nd** [comment]
No need to wait [anymore](https://forum.sublimetext.com/t/monokai-theme-for-sublime-merge-2/61646/2), @lzaaldian.

**Comment by lzaaldian** [comment]
> anymore

Finally! Thank you @bitsper2nd 🙌

**Comment by rorymullen** [comment]
Thank you!!!!!!

**Comment by bitsper2nd** [comment]
@lzaaldian @rorymullen I updated the theme.

**Comment by lzaaldian** [comment]
> @lzaaldian @rorymullen I updated the theme.

Awesome! Now others can see how it looks like before installing. I hope many people will like it 🙌
